src/main/java/ua/shpp/eqbot/cache/ServiceCache.java

```diff
@@ -24,6 +23,10 @@ public static void add(ServiceDTO serviceDTO) {
     public static void remove(ServiceDTO serviceDTO) {
         services.remove(serviceDTO.getTelegramId());
     }
+    public static void remove(Long id){
+        if(findBy(id) != null)
+            remove(findBy(id));
```

Comment:
```suggestion
        ServiceDTO serviceDTO = findBy(id);
        if(Objects.nonNull(serviceDTO))
            remove(serviceDTO);
```

What do you think about this?

Reply:
looks good, I will change